DRAFT.md

```diff
@@ -823,8 +821,8 @@ constexpr indirect(allocator_arg_t, const Allocator& a, indirect&& other)
 22. _Effects_: `alloc` is direct-non-list-initialized with `a`. If
     `other` is valueless, `*this` is valueless. Otherwise, if `alloc ==
     other.alloc` constructs an object of type `indirect` that owns the owned
-    value of other; `other` is valueless. Otherwise constructs an object of type
-    `indirect` using the specified allocator with `*other` used as an rvalue.
+    value of other; `other` is valueless. Otherwise constructs an owned object of type
+    `T` with `*other` used as an rvalue.
```

Comment:
```suggestion
    `T` with `*std::move(other)`.
```

This one is stylistic, doesn't really matter. It means the same thing either way. `std::move(*other)` would be the same too.